src/index.js

```diff
@@ -273,6 +290,7 @@ Dropzone.propTypes = {
 
   disablePreview: React.PropTypes.bool, // Enable/disable preview generation
   disableClick: React.PropTypes.bool, // Disallow clicking on the dropzone container to open file dialog
+  close: React.PropTypes.func,
```

Comment:
What's this prop is about? You can't tell from the name what it is supposed to do.